src/configure.ac

```diff
@@ -1458,61 +1380,48 @@ AC_MSG_CHECKING([whether to check for runtime dependencies])
 AC_MSG_RESULT($RUNTIME_CHECK)
 
 if test "$RUNTIME_CHECK" = "yes"; then
-AC_MSG_CHECKING([for BWidget using $TCLSH])
-if (unset DISPLAY; echo ["catch { package require bwidget }; exit [expr [lsearch [package names] BWidget] == -1]"] | $TCLSH); then
-    AC_MSG_RESULT([found])
-else
-    AC_MSG_RESULT(no)
-    AC_MSG_ERROR([BWidget not found!
-install with "sudo apt-get install bwidget"])
-fi
-
-AC_MSG_CHECKING([for BLT using tclsh8.5])
-if (test -f $TCL_EXEC_PREFIX/bin/tclsh8.5) && (echo ['if [catch {package require BLT}] { exit 1; } else { exit 0; }'] | tclsh8.5); then
-    AC_MSG_RESULT([found])
-else
-    AC_MSG_RESULT([not found])
-fi
-
-AC_MSG_CHECKING([for img::png using $TCLSH])
-if (unset DISPLAY; echo ["catch { package require img::png }; exit [expr [lsearch [package names] img::png] == -1]"] | $TCLSH); then
-    AC_MSG_RESULT([found])
-    HAVE_WORKING_BLT=yes
-else
-    AC_MSG_RESULT(no)
-    AC_MSG_ERROR([Tk img::png not found!
-install with "sudo apt-get install libtk-img"])
-fi
-
-AC_MSG_CHECKING([for tclX using $TCLSH])
-if (unset DISPLAY; echo ["catch { package require Tclx }; exit [expr [lsearch [package names] Tclx] == -1]"] | $TCLSH); then
-    AC_MSG_RESULT([found])
-else
-    AC_MSG_RESULT(no)
-    AC_MSG_ERROR([Tclx not found!
-install with "sudo apt-get install tclx"])
-fi
+    AC_MSG_CHECKING([for BWidget using $TCLSH])
+    if (unset DISPLAY; echo ["catch { package require bwidget }; exit [expr [lsearch [package names] BWidget] == -1]"] | $TCLSH); then
+        AC_MSG_RESULT([found])
+    else
+        AC_MSG_RESULT(no)
+        AC_MSG_ERROR([BWidget not found!
+    install with "sudo apt-get install bwidget"])
+    fi
 
-AC_MSG_CHECKING([for python pango and cairo modules])
-if $PYTHON -c 'import sys;raise SystemExit(sys.hexversion<0x3040000)'
-then
+    AC_MSG_CHECKING([for BLT using tclsh8.5])
```

Comment:
Shouldn't this use `$TCLSH` too?

Reply:
Modernized.  The reason was hidden deep in the git blame, something about ubuntu hardy :-/